### DISCUSSION TITLE
Dynamicaly change the variable label (name)

### DISCUSSION THREAD

**Discussion by Mateorid** [grafana/grafana#62580] [Dashboards, Panels & Visualizations]
An option to dynamically change the variable label as it's currently available for panel titles or alternatively include the variable label in the translation/localization of grafana.

What I'm trying to achieve is changing the variable label based on a different variable, something like this:
I have a query variable that would get the translated text
![image](https://user-images.githubusercontent.com/56208414/215760564-689c9982-121d-4dee-a244-56981deae67e.png)
That would be used similarly to the panel title 
![image](https://user-images.githubusercontent.com/56208414/215759543-792929fa-a883-4a3f-9359-67f2b6da9096.png)
And get the same result as with the panel
![image](https://user-images.githubusercontent.com/56208414/215759539-bafe171d-6fb6-4961-ba4a-38556731b91f.png)

Or would anyone be able to suggest a way to change the label in the current grafana?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is being addressed here https://github.com/grafana/grafana/issues/1032

If I am mistaken, feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
